.run/Main.run.xml

```diff
@@ -11,13 +11,13 @@
       <env name="TANAGRA_DB_INITIALIZE_ON_START" value="false" />
       <env name="TANAGRA_DB_USERNAME" value="dbuser" />
       <env name="TANAGRA_DB_PASSWORD" value="dbpwd" />
-      <env name="TANAGRA_UNDERLAY_FILES" value="broad/aou_synthetic/expanded/aou_synthetic.json,broad/cms_synpuf/expanded/cms_synpuf.json" />
+      <env name="TANAGRA_UNDERLAY_FILES" value="broad/aou_synthetic/expanded/aou_synthetic.json,broad/cms_synpuf/expanded/cms_synpuf.json,verily/sdd/expanded/sdd.json" />
       <env name="TANAGRA_FEATURE_ARTIFACT_STORAGE_ENABLED" value="true" />
       <env name="TANAGRA_AUTH_IAP_GKE_JWT" value="false" />
-      <env name="GOOGLE_APPLICATION_CREDENTIALS" value="$PROJECT_DIR$/rendered/tanagra_sa.json" />
+      <env name="GOOGLE_APPLICATION_CREDENTIALS" value="$PROJECT_DIR$/../Downloads/verily-tanagra-dev-d9842d6d2d76.json" />
```

Comment:
nit: swap local path with rendered one (the one generated by the `pull_credentials.sh` script)

Reply:
Reverted file